index.js

```diff
@@ -1,13 +1,33 @@
-
 var synth = require('synthetic-dom-events');
 
-var on = function(element, name, fn, capture) {
-    return element.addEventListener(name, fn, capture || false);
-};
+function isObject(item) {
+    return (Object.prototype.toString.call(item) == '[object Object]');
```

Comment:
The lodash version seems pretty extractable:

https://github.com/lodash/lodash/blob/9025c4a6571f65bb84abc25920b0365792ab561c/lodash.src.js#L8200-L8205

What do you think about plucking it out?